homeassistant/helpers/config_validation.py

```diff
@@ -973,6 +973,56 @@ def custom_serializer(schema: Any) -> Any:
     return voluptuous_serialize.UNSUPPORTED
 
 
+def expand_condition_shorthand(value: Any | None) -> Any:
+    """Expand boolean condition shorthand notations."""
+
+    if not isinstance(value, dict):
+        return value
+
+    if CONF_CONDITIONS not in value:
+        try:
```

Comment:
You've copy pasted the code 3 times. Iterate over the different options instead.

```python
for key, schema in (("and", AND_CONDITION_SHORTHAND_SCHEMA), ...):
```

Reply:
Ah, yes of course! For some reason I couldn't get out of my head that this couldn't be solved without `eval`...

Reply:
Python is a odd little fellow, isn't it @thomasloven 😄